app/src/main/kotlin/com/wire/android/ui/home/HomeViewModel.kt

```diff
@@ -42,6 +40,17 @@ class HomeViewModel @Inject constructor(
         viewModelScope.launch {
             loadUserAvatar()
         }
+        viewModelScope.launch {
```

Comment:
We can do it in the dame viewModelScope, and create a coroutine for each case
`
viewModelScope.launch {
            launch { loadUserAvatar() }
            launch { ongoingCalls... }
}
`